tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/pages/trusts/ofstedPage.ts

```diff
@@ -0,0 +1,149 @@
+            DateJoinedHeader: () => this.elements.currentRatings.Section().find("th:contains('Date joined')"),
+
+        }
+    };
+
+    public checkOfstedCurrentRatingsTableHeadersPresent(): this {
+        this.elements.currentRatings.SchoolNameHeader().should('be.visible');
+        this.elements.currentRatings.qualityOfEducationHeader().should('be.visible');
+        this.elements.currentRatings.behaviourAndAttitudesHeader().should('be.visible');
+        this.elements.currentRatings.personalDevelopmentHeader().should('be.visible');
+        this.elements.currentRatings.leadershipAndManagementHeader().should('be.visible');
+        this.elements.currentRatings.earlyYearsProvisionHeader().should('be.visible');
+        this.elements.currentRatings.sixthFormProvisionHeader().should('be.visible');
+        this.elements.currentRatings.beforeOrAfterJoiningHeader().scrollIntoView();
+        this.elements.currentRatings.beforeOrAfterJoiningHeader().should('be.visible');
+        return this;
+    }
+
+    public checkOfstedCurrentRatingsSorting(): this {
+        TableUtility.checkStringSorting(
+            this.elements.currentRatings.SchoolName,
+            this.elements.currentRatings.SchoolNameHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.currentRatings.qualityOfEducation,
+            this.elements.currentRatings.qualityOfEducationHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.currentRatings.behaviourAndAttitudes,
+            this.elements.currentRatings.behaviourAndAttitudesHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.currentRatings.personalDevelopment,
+            this.elements.currentRatings.personalDevelopmentHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.currentRatings.leadershipAndManagement,
+            this.elements.currentRatings.leadershipAndManagementHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.currentRatings.earlyYearsProvision,
+            this.elements.currentRatings.earlyYearsProvisionHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.currentRatings.sixthFormProvision,
+            this.elements.currentRatings.sixthFormProvisionHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.currentRatings.beforeOrAfterJoining,
+            this.elements.currentRatings.beforeOrAfterJoiningHeader
+        );
+        return this;
+    }
+
+    public checkQualityOfEducationJudgementsPresent(): this {
+        this.elements.currentRatings.qualityOfEducation().each((element) => {
+            const text = element.text().trim();
+            expect(text).to.match(/Good|Outstanding|No Judgement|Outstanding|Requires improvement|Inadequate|Not yet inspected/);
```

Comment:
```suggestion
            expect(text).to.match(/Good|Outstanding|No judgement|Outstanding|Requires improvement|Inadequate|Not yet inspected/);
```

Reply:
Already updated this with latest push - should now be resolved